docs/usage/installation.md

```diff
@@ -24,9 +24,25 @@ For other type of projects we recommend installing **semantic-release** directly
 $ npx semantic-release
 ```
 
-**Note**: For a global installation, it's recommended to specify the major **semantic-release** version to install (for example with `npx semantic-release@18`).
-This way your build will not automatically use the next major **semantic-release** release that could possibly break your build.
-You will have to upgrade manually when a new major version is released.
-
-**Note**: `npx` is a tool bundled with `npm@>=5.2.0`. It is used to conveniently install the semantic-release binary and to execute it.
-See [What is npx](../support/FAQ.md#what-is-npx) for more details.
+### Notes
+
+1. For a global installation, it's recommended to specify the major **semantic-release** version to install (for example with `npx semantic-release@18`).
+   This way your build will not automatically use the next major **semantic-release** release that could possibly break your build.
+   You will have to upgrade manually when a new major version is released.
```

Comment:
i have not tested the regex manager config suggested in this PR against an npx command that specifies only the major version. it is unclear if it would update only the major version, replace the partial definition with a full one, or fail in some way. is there a difference in this case that is worth clarifying in some way, or should both scenarios be expected to work with the suggested config?